exporters/otlp/all/src/main/java/io/opentelemetry/exporter/otlp/http/trace/OtlpHttpSpanExporterBuilder.java

```diff
@@ -93,13 +95,25 @@ public OtlpHttpSpanExporterBuilder setTrustedCertificates(byte[] trustedCertific
 
   /**
    * Sets ths client key and the certificate chain to use for verifying client when TLS is enabled.
-   * The key must be PKCS8, and both must be in PEM format.
+   * The key must be PKCS8, and both must be in PEM format. This will replace any other key manager
+   * that has been configured with this method or with setKeyManager().
    */
   public OtlpHttpSpanExporterBuilder setClientTls(byte[] privateKeyPem, byte[] certificatePem) {
     delegate.configureKeyManager(privateKeyPem, certificatePem);
     return this;
   }
 
+  /**
+   * Sets the SSLSocketFactory to use when TLS is enabled. When providing a preconfigured
+   * SSLSocketFactory, a trust manager must also be provided.
```

Comment:
The javadoc for this is different than the javadoc for the same method on `OtlpGrpcMetricExporterBuilder`. Let's make them consistent.